coral-schema/src/test/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverterTests.java

```diff
@@ -983,6 +983,19 @@ public void testComplexUnionType() {
     Assert.assertEquals(actualSchema.toString(true), TestUtils.loadSchema("testComplexUnionType-expected.avsc"));
   }
 
+  @Test
+  public void testUnionFixedAndBytes() {
+    String viewSql = "CREATE VIEW v AS SELECT b1.Fixed_field as c1 FROM basefixed b1" + " UNION ALL "
+        + "SELECT b2.Bytes_field as c1 FROM basebytes b2";
+
```

Comment:
There are still 2 lowercase `as`

Reply:
thanks, looks like it is restored during the rebase, fixed